force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,65 +87,123 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
-    }
+        String str = Null;
+        if (a == 0){
+            str = 'Zero';
+        } else if (a < 0){
+            str = 'Negative';
+        } else if (a > 0){
+            str = 'Positive';
+        }
+            return str;       
+            
+        }
```

Comment:
be sure you are checking for `null` value of `a`. What happens if you try to do `null < 0`?